### ISSUE TITLE
[Linux] Track-o-Bot compiles in qt4 and qt5. It runs and I can see the GUI; match results never show up on trackobot.com

### ISSUE THREAD

**Issue by gnummers** [BOSSoNe0013/track-o-bot#9]
Hi BOSSoNe0013,

Thanks for all the time you've taken to write a linux port. What information do you need for me to help puzzle out a fix? I can't quite figure out where I'm hanging up. Track-o-Bot GUI works fine. It can find my ~/.Hearthstone directory and see that Hearthstone is running. I was able to uncomment some LOG entries about window geometry and that function is working. But, results are never sent to trackobot.com, or at least they're not sent correctly. I haven't figured out how to get a proper debug version going yet to probe further. I've been spinning my wheels quite a bit trying to hack together a fix.

Please see:
http://redd.it/391n3j
https://github.com/stevschmid/track-o-bot/issues/81

http://i.imgur.com/L1TFLiC.png
http://i.imgur.com/uyGcw1K.png
http://i.imgur.com/gJaayyg.png

It seems others are having issues as well. I'm more than willing and available to test.

gnum@vincent-desktop ~ $ uname -a
Linux vincent-desktop 4.0.5-gentoo #1 SMP PREEMPT Sun Jun 7 21:27:28 EDT 2015 x86_64 Intel(R) Core(TM)2 Quad CPU Q9550 @ 2.83GHz GenuineIntel GNU/Linux

**Comment by gnummers** [comment]
Got it working! Still trying to wrap my mind around what the solution was, though... 

All I had to do was grab the master branch from BossOne's github[4] and edit...
#define LINUX_WINDOW_TITLE_BAR_HEIGHT 28
...to reflect what value was correct for my desktop environment.
Then:
qmake PREFIX=/usr track-o-bot.pro
make
make install
.
[02:17:50] Account bold-archmage-2706 found
[02:17:50] HS config file: /home/gnum/.Hearthstone/log.config
[02:20:52] Hearthstone found
[02:33:33] Upload ranked victory vs. priest as warrior. Went second
[02:33:34] Result was uploaded successfully!
[02:33:43] Hearthstone was closed
http://i.imgur.com/XQVtnkr.png [5]

**Comment by BOSSoNe0013** [comment]
Hi,
happy to see that's working for you. I'll investigate on this problem and see if i can get the title bar height programaticaly.
Thanks for your feedback :)

**Comment by gnummers** [comment]
And thank you!

I found another fix for you: http://trac.wildfiregames.com/ticket/2184

tl;dr it gets rid of the libpng warnings if you strip the pngs (for the icons, etc) with imagemagick.

**Comment by BOSSoNe0013** [comment]
Hi,
i've tested all png files in the project with methods found in your link and none seems to be problematic. I've got no warning while compiling or running the app.
Which file is problematic for you ?
Cheers
